4/Dockerfile

```diff
@@ -23,6 +23,7 @@ RUN wget -U "scannercli" -q -O /opt/sonar-scanner-cli.zip https://binaries.sonar
     && unzip sonar-scanner-cli.zip \
     && rm sonar-scanner-cli.zip \
     && mv sonar-scanner-${SONAR_SCANNER_VERSION}-linux ${SONAR_SCANNER_HOME} \
+    && chown --recursive scanner-cli:scanner-cli ${SONAR_SCANNER_HOME} \
```

Comment:
Why not allow mounting the cacerts file as volume? As far as I know it should be possible to mount a single file as volume, then used can simple create there own version of the cacerts file.

I'm also not sure how using the keytool would work when starting a scanner container that would first 'auto' import the certs required to allow use of self signed certs.